packages/design-system/src/components/Alert/Alert.jsx

```diff
@@ -51,10 +52,12 @@ export class Alert extends React.PureComponent {
             : '';
       }
 
-      sendAnalyticsEvent(
-        get(this.props.analytics, eventAction),
-        get(defaultAnalytics(this.eventHeadingText, this.props.variation), eventAction)
-      );
+      if (alertSendsAnalytics()) {
```

Comment:
Ah, just noticed - Ok if I move ` if (alertSendsAnalytics()) {` further up the compponentDidMount() function?

Reply:
Oh yep, thanks. I skimmed over that code above and didn't realize it was all for analytics. Good catch 👍